docs/kratos/passwordless/05_passkeys.mdx

```diff
@@ -78,7 +78,13 @@ import TabItem from '@theme/TabItem';
     passkey:
       enabled: true
       config:
-        display_name: "My Display Name"`}</CodeBlock>
+        rp:
+          display_name: Your Application name
```

Comment:
This is only needed when self-hosting Ory Kratos. In Ory Network we automatically set these values. Can you please clarify that, by - for example - adding an additional tab instead?

Reply:
Updated, please check :)